server/data/data-jpa/src/test/java/org/apache/james/rrt/jpa/JPARecipientRewriteTableTest.java

```diff
@@ -77,18 +78,18 @@ protected void addMapping(String user, Domain domain, String mapping, int type)
     }
 
     @Override
-    protected void removeMapping(String user, Domain domain, String mapping, int type) throws RecipientRewriteTableException {
+    protected void removeMapping(String user, Domain domain, String mapping, Type type) throws RecipientRewriteTableException {
         switch (type) {
-        case ERROR_TYPE:
+        case Error:
             virtualUserTable.removeErrorMapping(user, domain, mapping);
             break;
-        case REGEX_TYPE:
+        case Regex:
             virtualUserTable.removeRegexMapping(user, domain, mapping);
             break;
-        case ADDRESS_TYPE:
+        case Address:
             virtualUserTable.removeAddressMapping(user, domain, mapping);
             break;
-        case ALIASDOMAIN_TYPE:
+        case Domain:
             virtualUserTable.removeAliasDomainMapping(domain, Domain.of(mapping));
             break;
         default:

```

Comment:
missing asString in exception message concatenation

Reply:
Outdated.